### PR TITLE
More docs updates

### PR DESCRIPTION
**Pull request by rkevin-arch**

### CODE REVIEW

File: docs/api/authentication.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ POST Example
 vngshare Authentication
 -----------------------
 
-For ``vngshare``, supply the username to the GET param or POST data ``user``.
+For ``vngshare``, there is no authentication. The username is specified in the GET param or POST data field ``user``.
```
Comment (lxylxy123456) [inline docs/api/authentication.rst:36]:
```suggestion
For ``vngshare``, there is no password authentication. The username is specified in the GET param or POST data field ``user``.
```

File: docs/user_guide/notes_admin.rst [hunk 2]
```diff
@@ -14,6 +14,12 @@ Race Condition
 --------------
 ngshare should NOT be run concurrently, or there may be race conditions and data may be corrupted. For example, do not create multiple ngshare instances that share the same underlying database.
 
+Storage
+-------
+If you're using the Helm chart, only 1Gi of storage is allocated by default. You may increase this limit by specifying `pvc.storage` in the Helm values. If ngshare returns 500 for requests, lack of storage space could be a reason.
```
Comment (lxylxy123456) [inline docs/user_guide/notes_admin.rst:19]:
```suggestion
If you're using the Helm chart, only 1GiB of storage is allocated by default. You may increase this limit by specifying `pvc.storage` in the Helm values. If ngshare returns 500 for requests, lack of storage space could be a reason.
```

File: docs/user_guide/notes_admin.rst [hunk 3]
```diff
@@ -14,6 +14,12 @@ Race Condition
...
+
+Also, when courses are deleted, their corresponding files are not automatically deleted. You may want to delete these files to clean up storage.
```
Comment (lxylxy123456) [inline docs/user_guide/notes_admin.rst:21]:
```suggestion
Also, when courses or assignments are deleted, their corresponding files are not automatically deleted. You may want to delete these files to clean up storage.
```

File: docs/contributer_guide/project_structure.rst [hunk 4]
```diff
@@ -53,6 +53,19 @@ Deployment
 
 Testing
 -------
-``testing/`` is for installing ``ngshare`` in a Kubernetes setup.
+``testing/`` contains setups used for testing ngshare, ngshare_exchange nbgrader and Z2JH.
```
Comment (aalmanza1998) [inline docs/contributer_guide/project_structure.rst:56]:
```suggestion
``testing/`` contains setups used for testing ngshare, ngshare_exchange, nbgrader, and Z2JH.
```

File: docs/user_guide/notes_admin.rst (hunk 3 above)
Comment (lxylxy123456) [reply docs/user_guide/notes_admin.rst:21]: Removing Semantics

File: docs/contributer_guide/project_structure.rst (hunk 4 above)
Comment (Lawrence37) [inline docs/contributer_guide/project_structure.rst:56]: Typo?

File: docs/contributer_guide/project_structure.rst [hunk 5]
```diff
@@ -53,6 +53,19 @@ Deployment
...
+
+``testing/install_jhmanaged`` contains a Docker environment that demonstrates how a regular user would install ngshare and ngshare_exchange.
+
+``testing/install_z2jh`` contains a minikube environment that demonstrates how a regular user would install ngshare and ngshare_exchange on a standard Kubernetes cluster.
+
+``
+
+ngshare_exchange
+----------------
++
```
Comment (Lawrence37) [inline docs/contributer_guide/project_structure.rst:70]: What's this?

File: docs/contributer_guide/project_structure.rst [hunk 6]
```diff
@@ -53,6 +53,19 @@ Deployment
...
+
+``testing/docker`` contains a Docker environment for initial testing. It is slightly out of date and still uses our fork of ngshare rather than ngshare_exchange.
+
+``testing/minikube`` contains a minikube environment. This is the main testing setup for local development, and it uses ngshare and ngshare_exchange on the local filesystem.
+
+``testing/install_jhmanaged`` contains a Docker environment that demonstrates how a regular user would install ngshare and ngshare_exchange.
+
+``testing/install_z2jh`` contains a minikube environment that demonstrates how a regular user would install ngshare and ngshare_exchange on a standard Kubernetes cluster.
+
+``
```
Comment (aalmanza1998) [inline docs/contributer_guide/project_structure.rst:66]: What are these quotes?

**Pull request merged**